### PR TITLE
Evaluation for model-per-category script

### PR DESCRIPTION
**Pull request by jaor**
These patches add simple averaging evaluations to the model-per-category utilities.

### CODE REVIEW

File: model-per-category/evaluation/metadata.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Evaluation of category models",
+  "description": "Perform an ovaluation of per-category models",
```
Comment (mmerce) [inline model-per-category/evaluation/metadata.json]: typo: evaluation

File: model-per-category/evaluation/metadata.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "kind": "script",
+  "source_code": "script.whizzml",
+  "inputs": [
+    {
+      "name": "execution",
+      "type": "execution-id",
+      "description": "The execution of `Create category models` that created the per category models to evaluate."
```
Comment (mmerce) [inline model-per-category/evaluation/metadata.json]: typo: per-category

**Review by mmerce — COMMENTED** [review]
just two typos

**Comment by whizzmler** [comment]
typos fixed!

**Pull request merged**